docs/quickstart.md

```diff
@@ -0,0 +1,766 @@
+```bash
+sam init --runtime python3.9 --dependency-manager pip --app-template hello-world --name powertools-quickstart
+```
+If you want to play with the final code, then you can download it from [aws-samples](https://github.com/aws-samples/cookiecutter-aws-sam-python) github repository.
+=== "shell"
+```bash
+sam init --location https://github.com/aws-samples/cookiecutter-aws-sam-python
+```
+### Configuration
+If you deploy code from Quickstart make sure to [set up your AWS credentials](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-getting-started-set-up-credentials.html).
+
+### Code example
+In the `powertools-quickstart` folder, we modify the following files:
+
+* **app.py** - Application code.
+* **template.yaml** - AWS infrastructure configuration using SAM.
+* **requirements.txt** - List of extra Python packages needed.
+
+Let's configure our base application to look like the following code snippet.
+=== "app.py"
+
+    ```python
+    def hello():
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+    def lambda_handler(event, context):
+        return hello()
+    ```
+
+=== "template.yaml"
+
+    ```yaml
+    AWSTemplateFormatVersion: '2010-09-09'
+    Transform: AWS::Serverless-2016-10-31
+    Description: >
+        Sample SAM Template for powertools-quickstart
+    Globals:
+    Function:
+        Timeout: 3
+    Resources:
+    HelloWorldFunction:
+        Type: AWS::Serverless::Function
+        Properties:
+        CodeUri: hello_world/
+        Handler: app.lambda_handler
+        Runtime: python3.9
+        Architectures:
+            - x86_64
+        Events:
+            HelloWorld:
+            Type: Api 
+            Properties:
+                Path: /hello
+                Method: get
+    Outputs:
+        HelloWorldApi:
+            Description: "API Gateway endpoint URL for Prod stage for Hello World function"
+            Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/hello/"
+    ```
+Our lambda code consists of the `lambda_handler` method that is invoked by the API and the `hello` method that returns the results to the API gateway and is invoked by the handler itself.
+The SAM model configures API Gateway, which redirects traffic to Lambda for one path only: `hello`.
+!!! Warning 
+    For simplicity, we do not set up authentication and authorisation in the example!
+### Run your code
+At each point, you have two ways to run your code. Locally and within your AWS account. Given that we use SAM, the two methods are just as simple.
+#### Local test
+SAM allows you to execute a serverless application locally. Perform the next command in a shell.
+```bash
+> sam build && sam local start-api
+...
+2021-11-26 17:43:08  * Running on http://127.0.0.1:3000/ (Press CTRL+C to quit)
+```
+As a result API endpoint will be exposed for you. You can trigger it with the 'curl' command like in the following example.
```

Comment:
Tip: Not everyone might have curl installed (think Windows customers, unless they use WSL). Mention other forms like browser, or REST API clients.

Reply:
Added another tip:
```
???+ tip "Tip: Want to use different tool for API testing?"
    Instead of `curl` you have many other options to choose from.

    * Choose different command line tool like `HTTPie`.
    * Type the url in the browser directly
    * Use REST API client like `Postman` or `SoupUI`.
```